src/reconciler.tsx

```diff
@@ -115,8 +115,10 @@ export function applyProps(instance, newProps, oldProps = {}, interpolateArray =
     // Prep interaction handlers
     if (handlers.length) {
       // Add interactive object to central container
-      if (container && instance.raycast && !(handlers.length === 1 && handlers[0] === 'onUpdate'))
+      if (container && instance.raycast && !(handlers.length === 1 && handlers[0] === 'onUpdate')) {
         container.__interaction.push(instance)
+      }
+
       instance.__handlers = handlers.reduce(
```

Comment:
Private methods like `__handlers` these are a little hard to type, since in the official `three.js` types, these aren't included in the type definitions intentionally.

I think this is to discourage usage of these properties, but we could extend this types in `react-three-fiber`. Still need to find out how best to do that.